.changeset/fifty-apples-walk.md

```diff
@@ -0,0 +1,21 @@
+---
+"@builder.io/sdk-angular": major
+"@builder.io/sdk-react-nextjs": major
+"@builder.io/sdk-qwik": major
+"@builder.io/sdk-react": major
+"@builder.io/sdk-react-native": major
+"@builder.io/sdk-solid": major
+"@builder.io/sdk-svelte": major
+"@builder.io/sdk-vue": major
+---
+
+- BREAKING CHANGE :firecracker: : updated subscribeToEditor arguments:
+from:
+    subscribeToEditor('page', () => { ... }, options: {trustedHosts:['...']})
+to:
+    subscribeToEditor({
+        apiKey: '...',
+        model: '...',
+        trustedHosts: ['...'],
+        callback: () => { ... }
+    })
```

Comment:
Can you format this so the 🧨 appears properly and the code is in code blocks? 

Also, I'd explicitly state the two changes, which are that a) arguments are now passed as named parameters in object argument, and that b) `apiKey` is now a required field